src/corelibs/U2View/src/ov_assembly/AssemblyReadsAreaHint.cpp

```diff
@@ -66,20 +66,31 @@ AssemblyReadsAreaHint::AssemblyReadsAreaHint(QWidget* parent)
     setObjectName("AssemblyReadsAreaHint");
 }
 
-static QString getCigarString(const QString& ci) {
+static QString getCigarString(const QString& ci, bool trimmed = false) {
     if (ci.isEmpty()) {
         return QObject::tr("no information");
     }
 
     QString cigar;
-    for (int i = 0; i < ci.size(); ++i) {
+    int cigarSize = ci.size();
+    bool sizeBecameLess = false;
```

Comment:
rename to something more readable like 'isTrimmed'
Or remove it completely and use 
`if (cigarSize > AssemblyReadsAreaHint::LETTER_MAX_COUNT) cigar.append("...");`

Reply:
Renamed as `isTrimmed`. You fix won't work - `int cigarSize = qMin(cigarSize, AssemblyReadsAreaHint::LETTER_MAX_COUNT);`, which means, that `cigarSize` is always less or equal than `AssemblyReadsAreaHint::LETTER_MAX_COUNT`

Reply:
In my pseudo-code I by the `cigarSize` I meant size of the original cigar: `ci.size()`

Reply:
OK